app/views/users/show.html.slim

```diff
@@ -13,10 +14,10 @@
         .col-sm-4.col-sm-offset-4.pull-left.text-center
               = t('user.level')
               br
-              = "#{level}"
+              = "#{level.level}"
         .col-sm-4.col-sm-offset-4
           .progress
-            .progress-bar.progress-bar-striped.active aria-valuemax="100" aria-valuemin="0" aria-valuenow="45" role="progressbar" style=("width:  #{level*20}%") 
+            .progress-bar.progress-bar-striped.active aria-valuemax="#{level.to-level.from}" aria-valuemin="0"} aria-valuenow="45" role="progressbar" style=("width: #{confirmed_donations}%") 
```

Comment:
espacio alreadedor del "-" . Y ademas el Width ese no es porcentual, deberias hacer  (confirmed_donations - level.from)/(level.from - level.to)*100